### ISSUE TITLE
Build fails and old nugets, many dependency issues

### ISSUE THREAD

**Issue by papyr** [optimajet/DWKit#8]
Hi, the application kit project builds, but the starter application does not build. 
I updated to target to .Net core 2.1, and it resolved some issues.

**Comment by papyr** [comment]
![image](https://user-images.githubusercontent.com/12733414/48090926-3c8e2680-e1d6-11e8-98c5-6e7c006340cc.png)

**Comment by optimajet** [comment]
Dear Papyr.

We don't move to .NET Core 2.1. The task has been planned the next release.
Are messages from your screenshot warnings or errors?

**Comment by optimajet** [comment]
Can you share your experience? How did you fix these errors? Other of our customers didn't have them.

> but creating a new one did not save/show up
What does it mean? Do you suggest to add todo list in the admin panel?

> I am curious why did you choose react over ASP core/jquery. There is a much bigger audience/dev/customer base in the latter.
The main reason is react/redux is the most modern technology stack for web development.
We suppose many customers will migrate to react technology when they start new projects.

**Comment by optimajet** [comment]
Yes, we think about a wizard for doing main operations in DWKIT. This feature is in our backlog.
React/Redux is the better technology for creating rich web-apps. We did a big work for isolating users from complicated things. Basically, developers won't face react/redux if they don't drop into core-modules or source codes.

Coming soon we publish some video tutorials for each feature.

Anyway, thanks for your feedback. I hope you investigate the product successfully. If you decide to use DWKit in a commercial project, make sure our support service will help you.

**Comment by papyr** [comment]
Yes absolutely, I think there is a need, but its also big concern to recommend to company, because I will be on the hook to deliver and cannot fail. 
So for projects that are already on ASP core with JS/TS/Jquery, I have questions on integration.
Understanding the concepts, for e.g. applying your domain concepts, and then using the corresponding domains objects to do our specific internal work is also needed. Currently I have developed my own simple UI on stateless which is working so far. But for more flexibility and resuability I am considering your app. I honestly dont know the answers to all. But if you build out a simple good easy to understand sample on focus on that, you will get more traction. The other thing is the pricing is also not reflective of the ease of developer use, at least from my mgr. perspective. Just sharing feedback!
